tutorials/source-modeling/plot_forward.py

```diff
@@ -101,11 +101,15 @@
 # :func:`mne.setup_source_space`, while **volumetric** source space is computed
 # using :func:`mne.setup_volume_source_space`.
 #
-# We will now compute a surface-based source space with an ``'oct6'``
+# We will now compute a surface-based source space with an ``'oct4'``
 # resolution. See :ref:`setting_up_source_space` for details on source space
 # definition and spacing parameter.
+#
+# .. warn::
+#     ```'oct4'`` is used here just for speed, for real data the recommended
+#     spacing is ``'oct6'``.
```

Comment:
👍👍👍

Reply:
This was actually an errant push, will be part of another PR to fix Circle